cime_config/machines/config_machines.xml

```diff
@@ -2850,73 +2849,72 @@
       <cmd_path lang="sh">module</cmd_path>
       <cmd_path lang="csh">module</cmd_path>
       <modules>
         <command name="purge"/>
-        <command name="ls"/>
+        <command name="list"/>
         <command name="load">DefApps</command>
-        <command name="load">python/3.7.0</command>
-        <command name="load">subversion/1.9.3</command>
-        <command name="load">git/2.13.0</command>
+        <command name="load">python/3.7-anaconda3</command>
+        <command name="load">subversion/1.14.0</command>
+        <command name="load">git/2.31.1</command>
         <command name="load">cmake/3.20.2</command>
-        <command name="load">essl/6.1.0-2</command>
+        <command name="load">essl/6.3.0</command>
         <command name="load">netlib-lapack/3.8.0</command>
       </modules>
       <modules compiler="pgi.*">
-        <command name="load">pgi/19.9</command>
-        <command name="load">pgi-cxx14/default</command>
+        <command name="load">nvhpc</command>
       </modules>
       <modules compiler="pgigpu">
         <command name="load">cuda/10.1.243</command>
       </modules>
       <modules compiler="gnugpu">
-        <command name="load">cuda/10.1.105</command>
+        <command name="load">cuda/11.0.3</command>
       </modules>
       <modules compiler="ibm.*">
-        <command name="load">xl/16.1.1-9</command>
+        <command name="load">xl/16.1.1-10</command>
       </modules>
       <modules compiler="ibmgpu">
         <command name="load">cuda/10.1.243</command>
       </modules>
       <modules compiler="gnu">
-        <command name="load">gcc/8.1.1</command>
+        <command name="load">gcc/9.1.0</command>
       </modules>
       <modules compiler="gnugpu">
-        <command name="load">gcc/8.1.1</command>
+        <command name="load">gcc/9.1.0</command>
       </modules>
       <modules>
-        <command name="load">netcdf/4.6.1</command>
-        <command name="load">netcdf-fortran/4.4.4</command>
+        <command name="load">netcdf-c/4.8.0</command>
+        <command name="load">netcdf-fortran/4.4.5</command>
       </modules>
       <modules compiler="ibm" mpilib="!mpi-serial">
-        <command name="load">spectrum-mpi/10.3.1.2-20200121</command>
+        <command name="load">spectrum-mpi/10.4.0.3-20210112</command>
       </modules>
       <modules compiler="pgi.*" mpilib="!mpi-serial">
-        <command name="load">spectrum-mpi/10.3.1.2-20200121</command>
+        <command name="load">spectrum-mpi/10.4.0.3-20210112</command>
       </modules>
       <modules compiler="gnu" mpilib="!mpi-serial">
-        <command name="load">spectrum-mpi/10.3.1.2-20200121</command>
+        <command name="load">spectrum-mpi/10.4.0.3-20210112</command>
       </modules>
       <modules compiler="gnugpu" mpilib="!mpi-serial">
-        <command name="load">spectrum-mpi/10.3.1.2-20200121</command>
+        <command name="load">spectrum-mpi/10.4.0.3-20210112</command>
       </modules>
```

Comment:
Please combine into just single `<modules>` block: same mpi versions for all. Also, i don't think `mpi-serial` is used anywhere anymore.